prealloc.go

```diff
@@ -198,11 +198,22 @@ func exists(filename string) bool {
 	return err == nil
 }
 
+func contains(slice []string, item string) bool {
+	set := make(map[string]struct{}, len(slice))
```

Comment:
What is the point to create set here?
This should do same thing, am I missing something?
```
func contains(slice []string, item string) bool {
	for _, s := range slice {
		if s == item {
			return true
		}
	}

	return false
}
```

Reply:
Agreed, presumably we can just return if we find the element in question?

Reply:
true, fixed the function. 👍